src/raftstore/store/util.rs

```diff
@@ -89,6 +90,39 @@ pub fn conf_change_type_str(conf_type: &eraftpb::ConfChangeType) -> &'static str
     }
 }
 
+// Use delete range to delete all data in [start_key, end_key) for each column family.
+pub fn delete_all_in_range(db: &DB, start_key: &[u8], end_key: &[u8]) -> Result<()> {
+    if start_key >= end_key {
+        return Ok(());
+    }
+
+    let wb = WriteBatch::new();
+    for cf in db.cf_names() {
+        let iter_opt = IterOption::new(Some(end_key.to_vec()), false);
+        let mut it = box_try!(db.new_iterator_cf(cf, iter_opt));
+        it.seek(start_key.into());
+        if it.valid() {
+            let handle = box_try!(rocksdb_util::get_cf_handle(db, cf));
+            if cf == CF_WRITE {
+                // We enable memtable prefix bloom for CF_WRITE, for delete_range operation
+                // RocksDB will add start key to bloom, and the start key will go through
+                // function prefix_extractor->Transform, in our case the prefix_extractor is
+                // FixedSuffixSliceTransform, if the length of start key less than 8, we
+                // will encounter index out of range error.
+                box_try!(wb.delete_range_cf(handle, it.key(), end_key));
```

Comment:
Is it possible to let bloom filter to be compatible with this?

Reply:
What happen if `it.key() >= end_key`?

Reply:
it.key must > end_key, because end_key is upper_bound when we create the iterator